packages/flutter_tools/test/general.shard/cache_test.dart

```diff
@@ -179,3 +258,4 @@ class MockRandomAccessFile extends Mock implements RandomAccessFile {}
 class MockCachedArtifact extends Mock implements CachedArtifact {}
 class MockInternetAddress extends Mock implements InternetAddress {}
 class MockCache extends Mock implements Cache {}
+class MockOperatingSystemUtils extends Mock implements OperatingSystemUtils {}
```

Comment:
`context.dart` has one of these already

Reply:
actually there's a bunch of different files that define this class in different ways

Reply:
The ones that are using mockito are doing so purposefully in ways the basic fake class don't meet.

Reply:
presumably we only need two, though...